src/system/RenderableSystem.lua

```diff
@@ -20,6 +20,27 @@ function RenderableSystem:setScene ( tag )
 
 end
 
+function RenderableSystem:update( dt )
```

Comment:
In any case, all of this probably falls out of the scope of this particular pull request and issue